services/sysinfo/client/client.go

```diff
@@ -187,3 +193,159 @@ func (p *dmesgCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...interfa
 	}
 	return exit
 }
+
+type journalCmd struct {
+	since  string
+	until  string
+	tail   uint64
+	unit   string
+	output string
+}
+
+func (*journalCmd) Name() string     { return "journalctl" }
+func (*journalCmd) Synopsis() string { return "Get the log entries stored in journald" }
+func (*journalCmd) Usage() string {
+	return `journalCtl [--since|--S=X] [--until|-U=X] [-tail=X] [-u|-unit=X] [-o|--output=X]9 :
+	Get the log entries stored in journald by systemd-journald.service 
+
+	Note: we have turned off the way to display all journal entries since boot since that will exceed the default buffer limit set in sansshell.
```

Comment:
`since boot since` -> `since boot because` might read better? You can change because to due to or whatever else you want as well